javascript/javascript1/week4/task_1.js

```diff
@@ -0,0 +1,86 @@
+let myName = "";
+let todo = [];
+
+function getReply(command) {
+  const array = command.split(" ");
+  let response="";
+  if (command.includes("hello my name is")) {
+    myName = array[4];
+    response = `nice to see you ${myName}`;
```

Comment:
You should also consider if the user sends 'hello' twice with the same name as the task asks. Print something else then.